test/Driver/sanitize_stable_abi.swift

```diff
@@ -0,0 +1,5 @@
+// REQUIRES: asan_runtime
+// RUN: %swiftc_driver -driver-print-jobs -sanitize=address -sanitize-stable-abi  %s 2>&1 | %FileCheck %s
```

Comment:
When I read "-sanitize-stable-abi` as "sanitize stable ABI" it's not the right meaning.  Compare "Sanitize address" which describes pretty well what it does.

The variable names for this are `SanitizerUseStableABI`, so we just forgot the `r` here?

I think the "sanitizer" prefix here is a noun vs. "sanitize" (verb).  If we want to include a verb, we could spell the option with "use".

I would pick one of these:
```
-sanitizer-stable-abi
-sanitizer-use-stable-abi
```